tests/integration/targets/aci_switch_policy_vpc_protection_group/tasks/main.yml

```diff
@@ -197,7 +197,7 @@
   cisco.aci.aci_switch_policy_vpc_protection_group:
     <<: *aci_switch_policy_vpc_protection_group_query
     protection_group: ansible_test
-  check_mode: yes
+  check_mode: true
   register: cm_query_non_vpc_prot_grp
 
 - name: Query non-existing vpc protection group (normal mode)
```

Comment:
in aci_syslog_group ( including here because test file not change thus cannot comment ) 
```
include_ms: yes
include_time_zone: yes

Reply:
Please  also check the python module code

Reply:
resolved